keepassxc-browser/background/page.js

```diff
@@ -153,6 +158,23 @@ page.initOpenedTabs = async function() {
     }
 };
 
+page.initSitePreferences = async function() {
+    if (!page.settings) {
+        return;
+    }
+
+    // Delete previously created Object if it exists. It will be replaced by an Array
+    if (page.settings['sitePreferences'] !== undefined && page.settings['sitePreferences'].constructor === Object) {
+        delete page.settings['sitePreferences'];
+    }
+
+    if (!page.settings['sitePreferences']) {
+        page.settings['sitePreferences'] = [];
+    }
```

Comment:
This sequence seems excessive, shouldn't it be sufficient to just assign the variable? Garbage collection should take care of the previous object?

Reply:
True.

Reply:
I think the check at the beginning can be already removed. It was necessary maybe a year ago (?) when we switched Site Preferences from Object to an Array.